android/src/main/java/io/mosip/tuvali/ble/central/state/StateHandler.kt

```diff
@@ -256,7 +256,7 @@ class StateHandler(
     try {
       super.dispatchMessage(msg)
     } catch (e: Exception) {
-      listener.onException(StateHandlerException("Exception in Central State Handler", e))
+      listener.onException(WalletStateHandlerException("Exception in Central State Handler", e))
```

Comment:
If I think about it.

Wallet is higher layer than Central layer. So when Central layer is throwing and Exception. The Exception should be `CentralStateHandlerException`. Also the exception should be defined within `ble` package.

Same thing should apply for VerifierStateHandlerException.